src/UserSpaceInstrumentation/AllocateInTraceeTest.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <absl/strings/str_split.h>
+#include <gtest/gtest.h>
+#include <sys/ptrace.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+#include <csignal>
+#include <filesystem>
+#include <string>
+#include <vector>
+
+#include "AllocateInTracee.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "OrbitBase/WriteStringToFile.h"
+#include "UserSpaceInstrumentation/Attach.h"
+
+namespace orbit_user_space_instrumentation {
+
+namespace {
+
+using orbit_base::ReadFileToString;
+using orbit_base::WriteStringToFile;
```

Comment:
Do you still need the `#include` and `using` for `WriteStringToFile`?

Reply:
done